source/getting-started/presence-detection.markdown

```diff
@@ -18,14 +18,15 @@ It's also possible to run an app on your phone to provide detailed location info
 
 During the setup of Home Assistant Companion on your mobile device, the app will ask for permission to allow the device's location to be provided to Home Assistant. Allowing this will create a `device_tracker` entity for that device which can be used in automations and conditions.
 
-
 ### Zones
+
 ![Map with zones](/images/screenshots/badges-zone.png)
 
 {% term Zones %} allow you to name areas on a map. These areas can then be used to name the location a tracked user is, or use entering/leaving a zone as an automation {% term trigger %} or {% term condition %}. See [Zones integration](/integrations/zone/) page for more details like creating zones.
 
 <div class='note'>
-The map view will hide all devices that are home.
+The <a href="/dashboards/map/">map dashboard
+        </a>[map dashboard](/dashboards/map/) will hide all devices that are home.
```

Comment:
Why not on a single line? 

If you keep a blank line between your content and the HTML tags, you could also place a URL in markdown style.

Reply:
thank you for the catch!